test/test_pullparser.rb

```diff
@@ -82,6 +82,22 @@ def test_character_references
       assert_equal("B", events['b'])
     end
 
+    def test_predefined_entity_replacement
```

Comment:
```suggestion
    def test_text_entity_references
```

And could you rename the existing `test_character_references` to `test_text_character_references`?

Reply:
OK
I see.